### ISSUE TITLE
Maccy hides in full-screen with a notch

### ISSUE THREAD

**Issue by RonishJariwala** [p0deje/Maccy#637]
Chrome Version: Version 117.0.5938.149
macOS Sonoma 14.0
Maccy Version: Version 0.27.1 (22)

video:
https://github.com/p0deje/Maccy/assets/137525345/27d973d7-4655-43b5-ac61-e866306af749

Issue: 
I am writing to let you know that after the latest macOS update, the app is not working properly in full screen mode with Chrome or any other browser. We are unable to open MAccy when there is a maximized window open.

I’ve attached the video, it might help. So, when I tries to take my arrow down, it disappears. I’ve tried to showcase that with Shottr.

**Comment by p0deje** [comment]
Note that this only happens on MacBook with a notch where the menubar hides/shows upon movement of the mouse.

**Comment by p0deje** [comment]
Please open Maccy using a global shortcut (such as ⇧⌘C) as a workaround.

**Comment by p0deje** [comment]
This is fixed in https://github.com/p0deje/Maccy/commit/84422077275fc2f5e6f17b85c686e426af731e49, will be available in the next release unless this commit introduced some regressions.

**Comment by qodunpob** [comment]
Hi, @p0deje, the fix is seemed to be released but the issue hasn't gone yet

**Comment by p0deje** [comment]
@qodunpob Can you please provide a screen recording showcasing the issue?

**Comment by qodunpob** [comment]
@p0deje sure
https://github.com/p0deje/Maccy/assets/1608699/1bb28dac-ed35-4268-a5f7-da066a775692

**Comment by p0deje** [comment]
@qodunpob What macOS and Maccy versions do you run? Does it happen in for all apps?

**Comment by qodunpob** [comment]
@p0deje it does
macOS Sonoma 14.4
Maccy 0.30.0

**Comment by p0deje** [comment]
@qodunpob Do you use a Macbook with a notch or it happens on external display?

Also, please provide an output of running `defaults read org.p0deje.Maccy` in Terminal.app.

**Comment by qodunpob** [comment]
@p0deje I'm using MacBook Air M2 with notch, not an external display.
Here is an output
```
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":35}";
    "KeyboardShortcuts_popup" = "{\\"carbonKeyCode\\":9,\\"carbonModifiers\\":768}";
    "LaunchAtLogin__hasMigrated" = 1;
    NSNavLastRootDirectory = "/Applications";
    NSNavPanelExpandedSizeForOpenMode = "{800, 448}";
    "NSStatusItem Preferred Position Item-0" = 474;
    "NSStatusItem Visible Item-0" = 1;
    "NSWindow Frame NSNavPanelAutosaveName" = "335 369 800 448 0 0 1470 919 ";
    "NSWindow Frame com.sindresorhus.Preferences.FrameAutosaveName" = "464 222 442 493 0 0 1470 919 ";
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "464 431 542 380 0 0 1470 919 ";
    enabledPasteboardTypes =     (
        "public.file-url",
        "public.png",
        "public.html",
        "public.rtf",
        "public.utf8-plain-text",
        "public.tiff"
    );
    historySize = 20;
    ignoredApps =     (
        "com.bitwarden.desktop",
        "com.apple.systempreferences"
    );
    ignoredPasteboardTypes =     (
        "de.petermaurer.TransientPasteboardType",
        "net.antelle.keeweb",
        "Pasteboard generator type",
        "com.agilebits.onepassword",
        "com.typeit4me.clipping"
    );
    migrations =     {
        "2020-04-25-allow-custom-ignored-types" = 1;
        "2020-06-19-use-keyboardshortcuts" = 1;
        "2020-09-01-ignore-keeweb" = 1;
        "2021-02-20-allow-to-customize-supported-types" = 1;
        "2021-06-28-add-title-to-history-item" = 1;
        "2021-10-16-remove-dynamic-pasteboard-types" = 1;
        "2022-08-01-rename-suppress-clear-alert" = 1;
        "2022-11-14-add-html-rtf-to-supported-types" = 1;
        "2023-01-22-add-regexp-search-mode" = 1;
    };
    numberOfUsages = 48;
    pasteByDefault = 1;
    showInStatusBar = 1;
}
```

**Comment by p0deje** [comment]
Ah, it hides when you move a cursor, I'll see if there is a way to fix this.

**Comment by p0deje** [comment]
Fixed in 2.0
